src/components/OidcClient.js

```diff
@@ -25,6 +25,11 @@ class OidcClient {
         queryParams.append('code_challenge', codeChallenge)
         queryParams.append('redirect_uri', config.redirectUri)
 
+        const windowQueryParameter = new URLSearchParams(window.location.search);
+        if (windowQueryParameter.has('acr_values')) {
+            queryParams.append('acr_values', windowQueryParameter.get('acr_values'))
```

Comment:
shouldn't you add `acr` parameter as well?

Reply:
Why should I? This solution works well

Reply:
I was looking at curity-server code and in one place it expects `acr` query paramter with some value. That's why I thought it should be added manually.

Reply:
`acr_values` parameter is passed to the Curity server at the beginning of the authentication flow. Then curity server sends the client by using `redirect-response` to call the Curity server with the `acr` parameter.